pkg/fanal/analyzer/pkg/dpkg/copyright.go

```diff
@@ -130,3 +141,16 @@ func (a dpkgLicenseAnalyzer) Type() analyzer.Type {
 func (a dpkgLicenseAnalyzer) Version() int {
 	return dpkgLicenseAnalyzerVersion
 }
+
+func licensesSeparation(line string) []string {
+	line = strings.ReplaceAll(line, ",", "")      // e.g. GPL-1+ or Artistic, and BSD-4-clause-POWERDOG
```

Comment:
It looks flaky.

`GPL-1+,GPL-2` would be `GPL-1+GPL-2`, right?

Reply:
Fixed